docker/sparkswapd/Dockerfile

```diff
@@ -39,6 +42,15 @@ COPY --from=builder /home/app/proto /home/app/proto
 
 # Copy node_modules
 COPY --from=builder /home/app/node_modules /home/app/node_modules
+COPY --from=builder /home/app/broker-cli/node_modules /home/app/broker-cli/node_modules
+
+ARG USER=root
```

Comment:
I think this should be a path and the name should reflect that, e.g. `USER_HOME=/root`